src/view/com/posts/FeedItem.tsx

```diff
@@ -304,14 +305,14 @@ let FeedItemInner = ({
               />
               <Text
                 type="sm-bold"
-                style={pal.textLight}
+                style={[pal.textLight, {display: 'flex', flexDirection: 'row'}]}
                 lineHeight={1.2}
                 numberOfLines={1}>
                 {isOwner ? (
                   <Trans>Reposted by you</Trans>
                 ) : (
                   <Trans>
-                    Reposted by{' '}
+                    Reposted by{NON_BREAKING_SPACE}
```

Comment:
Maybe it's worth adding a comment to the two locations to explain the change, since these are the only two places where nbsp is used within localized messages?